app/src/main/res/layout/fragment_plant_detail.xml

```diff
@@ -82,6 +82,28 @@
             app:layout_constraintEnd_toEndOf="@id/guideline_end"
             app:layout_constraintTop_toBottomOf="@id/tv_title_tasks"/>
 
+        <TextView
+            android:id="@+id/tv_plant_gallery_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_16dp"
+            android:text="@string/title_plant_gallery_text"
+            android:textSize="@dimen/text_size_normal"
+            android:visibility="gone"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintEnd_toEndOf="@id/guideline_end"
+            app:layout_constraintTop_toBottomOf="@id/rv_detail_tasks"/>
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/rv_plant_photos"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_8dp"
+            android:visibility="gone"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/tv_plant_gallery_text"/>
+
```

Comment:
How about move it into some container to set visibility=gone only on one container